### ISSUE TITLE
amqp-transport does auto setup for each get/publish message

### ISSUE THREAD

**Issue by TheHett** [symfony/symfony#39605]
**Symfony version(s) affected**: 5.2

**Description**  
The transport `Symfony\Component\Messenger\Bridge\Amqp\Transport\Connection` trying to create and declare queues and exchanges for each publish/get message. I suspect it's not necessary, it is enough to do it only once.

https://github.com/symfony/amqp-messenger/blob/a7f681b022f9cfb88febc12face05b12a47232c1/Transport/Connection.php#L422

**Possible Solution**  
I think it's need to add a flag that the setup already done and do not call it again.

_Sorry for my english._

**Comment by TheHett** [comment]
This leads to a decrease the performance and an increase the load on rmq.

**Comment by TheHett** [comment]
Thanks!
